### ISSUE TITLE
Support for multiple configurations? 

### ISSUE THREAD

**Issue by mincej** [gagneurlab/drop#457]
Hello, I was wondering if there is support for multiple configuration files. 
I wanted to test out different parameters, and as such I have multiple configuration files stored in `PATH/Configurations`. Each of these files has different corresponding sample annotations and output/report paths. 

When I try to run snakemake using `snakemake --cores 6 --configfile ./Configurations/config1.yaml`, I get a MissingRuleException error. However, if I copy the configuration file out of the `Configurations` directory, rename it to `config.yaml` in the same directory as `Snakefile` and `Scripts`, then it works when running `snakemake --cores 6`, even though no change was made to the configuration file itself. Is there support for the the `--configfile` option that snakemake provides? Would I need to make a separate directory and `drop init` each one for each of the test configurations that I wish to test?

**Comment by vyepez88** [comment]
Hi, I have tried DROP with different config files using the `--configfile` option and it worked. I only tried the sampleQC module, however. I would suggest to have different directories each containing its own config file and specifying different `root` directories.

**Comment by mincej** [comment]
Hey Vicente, that's what I ended up trying, but here's what I ultimately decided to do: 
I have 4 groups of 5 samples, each group preprocessed differently. 
I also use 50 external samples, each of which helps boost the sample size of each of the four groups. 

I put these all together into one sampleAnnotation and one configuration file, and am analyzing the results on a group-by-group basis. As an example, I have 4 OUTRIDER groups, 1 of each assigned to each of the unique groups of local samples and every group assigned to the external samples. 

Otherwise, I figured my original strategy didn't work due to the .drop and .snakemake directories that were getting overwritten and different locks. In the future I'll be either using the strategy outlined above or what you had recommended with different root directories!
